addons/alk/README.md

```diff
@@ -0,0 +1,5 @@
+# alk
+Arm Leg Kit. Allows medics to fix broken limbs without using a PAK.
+
+## Maintainer(s)
+* derZade
```

Comment:
:thinking: we could write this `@DerZade`  and have the user linked when watching the file on github.